### DISCUSSION TITLE
This site can’t be reached

### DISCUSSION THREAD

**Discussion by vivekarora56** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#787] [Q&A]
Tried everything twice. Getting stuck on last step.

![image](https://user-images.githubusercontent.com/88108089/195337716-c46b74c1-d651-45d8-aae7-aa7af7a08d10.png)

**Comment by im-coder-lg** [comment]
From the url copy the contents between `?code=` and `?scope` and paste it as your code, it will work.

**Reply by BaileyKlemmensen** [reply]
what url?

**Reply by im-coder-lg** [reply]
That auth url

**Reply by joemacd117** [reply]
![image](https://user-images.githubusercontent.com/124471999/216821132-ba64cff8-1fde-405c-884a-ba3068278a7e.png)


This one, I had the same issue, the above comment solved it for me.
